simple-local-avatars.dev.js

```diff
@@ -77,6 +77,33 @@ jQuery(document).ready(function($){
 			avatar_preview.attr( 'src', current_avatar );
 		}
 	} );
+
+	$( document.getElementById('simple-local-avatars-migrate-from-wp-user-avatar') ).on( 'click', function(event) {
+		event.preventDefault();
+		jQuery.post( ajaxurl, { action: 'migrate_from_wp_user_avatar', migrateFromWpUserAvatarNonce: i10n_SimpleLocalAvatars.migrateFromWpUserAvatarNonce } )	
```

Comment:
Does this need to be `jQuery` here or can it just use `$`? Real question, I haven't traced through to see if I missed something in the scope.

Reply:
This matches how another ajax request is being made on line 38. Most likely all of the instances of `jQuery` can be `$` since that parameter is being passed in on line 12. Leaving as is for now to match the current pattern.